.golangci.yml

```
@@ -82,6 +88,14 @@ linters:
         path: _test\.go
       - path: pkg/resolve/resolve.go
         text: don't use `init` function
+      - linters:
+          - misspell
+        path: pkg/provider/gitlab/gitlab\.go
+        text: '`canceled` is a misspelling of `cancelled`'
```

```suggestion
        text: 'found `canceled` it must be `cancelled`'
```

you can't say that its misspelled as its just a other dialect of English some might choose to write 🙂

---

maybe better:

```
use american english version of the word `cancelled` as `canceled` with one L
```

---

@zakisk this text field in the exclusion rule is not a custom message, it needs to exactly match the error message generated by the misspell linter itself. This is because golangci-lint uses it to pattern match against the linter's output when applying exclusions. In this case, I'm trying to exclude the files related to GitLab.

---

ah, yeah I didn't know that but in that can you can suppress the lint on CreateStatus func in gitlab.go using comment on it `//nolint:misspell`

---

Thanks, @zakisk! I will suppress the lint just for that function.